k6/increaseSeats.test.js

```
@@ -0,0 +1,28 @@
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+
+export let options = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'options' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)